common/vehicle_constants_manager/package.xml

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0"?>
+<?xml-model href="http://download.ros.org/schema/package_format3.xsd" schematypens="http://www.w3.org/2001/XMLSchema"?>
+<package format="3">
+  <name>vehicle_constants_manager</name>
+  <version>1.0.0</version>
+  <description>Library package to help nodes access vehicle specific constants.</description>
+  <maintainer email="mfc@leodrive.ai">M. Fatih Cırıt</maintainer>
+  <license>Apache 2.0</license>
+
+  <buildtool_depend>ament_cmake_auto</buildtool_depend>
+  <buildtool_depend>autoware_auto_cmake</buildtool_depend>
+
+  <depend>autoware_auto_common</depend>
+  <depend>rclcpp</depend>
+  <depend>rclcpp_components</depend>
+
+  <test_depend>ament_cmake_gtest</test_depend>
+  <test_depend>ament_lint_common</test_depend> -->
```

Comment:
```suggestion
  <test_depend>ament_lint_common</test_depend> 
```

Reply:
I'm working on fixing the pre-commit now.
I'll fix it there at the same time. :bow:

Reply:
fixed https://github.com/autowarefoundation/autoware.universe/pull/185/commits/50ab9b5e44fbe83fc3a4c898341dfd0a7a00fe27